### ISSUE TITLE
Why I am getting 'ERROR reincarnated worker...' without any error specifics?

### ISSUE THREAD

**Issue by igalci** [Koed00/django-q#711]
I have a function that works fine manually, but in DjangoQ it returns `ERROR reincarnated worker Process-1:534 after death` over and over again without giving me any specifics of what the error was?

So my question is, in what case will DjangoQ just destory the worker and not specify what error caused it?

More context, I run Django-Q in an AWS EC2, and my only suspicion is that my EC2 runs out of memory or resources during this task, and DjangoQ has no chance to explain what the error was.

Can anyone confirm? is running out of memory a reason DjangoQ will just quit like above without showing any exceptions?

**Comment by tudoanh** [comment]
Can you resolve this issue?

**Comment by igalci** [comment]
Yes, this happens when the instance runs out of memory. It just shuts down the process without any explicit errors. Make sure you keep the functions efficient and below the MAX memory usage.

This was my case.
